experimental/GroebnerWalk/src/common.jl

```diff
@@ -0,0 +1,161 @@
+
+@doc raw"""
+    groebner_walk(
+      I::MPolyIdeal, 
+      target::MonomialOrdering = lex(base_ring(I)),
+      start::MonomialOrdering = default_ordering(base_ring(I));
+      perturbation_degree = length(gens(base_ring(I))),
+      algorithm::Symbol = :standard
+    )
+
+Compute a reduced Groebner basis w.r.t. to a monomial ordering by converting it using the Groebner Walk.
+The Groebner Walk is proposed by Collart, Kalkbrener & Mall (1997).
+One can choose a strategy of:
+- Standard Walk (:standard) computes the Walk like as presented in Cox, Little & O'Shea (2005).
+- Generic Walk (:generic) computes the Walk as presented in Fukuda, Jensen, Lauritzen & Thomas (2005).
+- Perturbed Walk (:perturbed, with p = degree of the perturbation) computes the Walk as presented in Amrhein, Gloor & Küchlin (1997).
```

Comment:
this looks a bit redundant to the algorithm keyword argument explanation below. Maybe these two lists can be merged?